ocs_ci/ocs/acm/acm.py

```diff
@@ -513,6 +514,20 @@ def import_clusters_via_cli(clusters):
             column="JOINED",
             resource_name=cluster[0],
         )
+
+        log.info("Create the Klusterlet addon configuration")
+        klusterlet_config = templating.load_yaml(constants.ACM_HUB_KLUSTERLET_YAML)
+        klusterlet_config["metadata"]["name"] = cluster[0]
+        klusterlet_config["metadata"]["namespace"] = cluster[0]
+        klusterlet_config_obj = OCS(**klusterlet_config)
+        klusterlet_config_obj.apply(**klusterlet_config)
```

Comment:
are we editing the existing resource OR creating a new one ?

Reply:
Creating new, made the change.